simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -42,25 +42,45 @@ export const HowToCreate = () => (
           <ul className={styles['list-info']}>
             <img src="/images/wand.svg" alt="Wand" />
             <li>
-              <h3>Create Queue</h3>
+              <h3>1. Create Queue</h3>
               <p>
                 Enter queue name of your choice and create a brand new queue totally free of cost
               </p>
             </li>
             <img src="/images/share-24px.svg" alt="Share" />
             <li>
-              <h3>Share queue</h3>
-              <p>Click the share button and send the queue link to your users for them to join</p>
+              <h3>2. Share Queue</h3>
+              <p>
+                Click the share button and send the queue link to your users for them to join. Or
+                print the QR code and stick it at a place where people can scan it.
+              </p>
             </li>
             <img src="/images/people-24px.svg" alt="People" />
             <li>
-              <h3>Manage queue</h3>
+              <h3>3. Manage queue</h3>
               <p>
                 Manage your queues seamlessly and efficiently to provide a better user experience
               </p>
             </li>
           </ul>
         </div>
+        <p>
+          SimplQ customer waiting experience software is a simple and affordable ticket, table or
+          place in line solution, adding value to any crowded business.
+        </p>
```

Comment:
Let's say place-in-line? Also, how about saying business involving waiting crowds?

Reply:
I've made it `place-in-line`. "crowded business" seems to be a good usage to me that "business involving waiting crowds".